### PR TITLE
sci-lib/blis: use python-any-r1 eclass instead of hardcoded python deps.

### PR DESCRIPTION
**Pull request by cdluminate**
Closes: https://bugs.gentoo.org/693130

### CODE REVIEW

**Comment by cdluminate** [comment]
Repoman result clean. @heroxbd

```
$ repoman full -dx

RepoMan scours the neighborhood...
  ebuild.minorsyn               1
   sci-libs/blis/blis-0.5.2.ebuild: Ebuild contains leading spaces on line: 19
RepoMan sez: "You're only giving me a partial QA payment?
              I'll take it this time, but I'm not happy."
```

File: sci-libs/blis/blis-0.5.2.ebuild [hunk 1]
```diff
@@ -2,7 +2,8 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-inherit eutils
+PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} )
+inherit eutils python-any-r1
```
Comment (mgorny) [inline sci-libs/blis/blis-0.5.2.ebuild]: Where is eutils used?

Comment (cdluminate) [reply sci-libs/blis/blis-0.5.2.ebuild]: This package invokes python for generating a header file, so we only need the `${PYTHON_DEPS}` variable from the eclass. The python eclass has nothing to do with the rest part of the package.

Comment (cdluminate) [reply sci-libs/blis/blis-0.5.2.ebuild]: Oops, I misread the question. `eutils` is not used here. It's sort of leftover from the template I used.

**Comment by cdluminate** [comment]
@mgorny Fixed the problems and force-pushed.

**Comment by heroxbd** [comment]
Thanks Michal.

**Pull request closed without merging**